idle-rpg/game/utils/Map.js

```diff
@@ -5,10 +5,27 @@ class Map {
 
   moveToRandomMap(selectedPlayer) {
     const movement = Helper.randomBetween(0, 100);
+    let movementSpeed = 1;
+    if ( selectedPlayer.horse === "Yes" ){
+      let movementSpeed = randomBetween (1,3);
+      
```

Comment:
If we store the horse data as an object inside equipments, we can store the movement speed into the player obj that is saved in the database, also when you intiailized movementSpeed with the let above, you dont have to initialize it again inside the if statement